vscode-lean4/test/suite/bootstrap/bootstrap.test.ts

```diff
@@ -70,43 +74,4 @@ suite('Lean4 Bootstrap Test Suite', () => {
 
     }).timeout(600000);
 
-    test('Create linked toolchain named master', async () => {
```

Comment:
this test was always a bit bogus because it is literally copying a known good install, this is very different from actually using a locally build binaries from the lean4 repo, unfortunately testing that using a github artifact turned out to be unreliable, so we'll leave that one to manual testing for now.